argoverse/sensor_dataset_config.py

```diff
@@ -49,8 +47,27 @@ class SensorDatasetConfig:
     camera_sensors: SensorSuiteConfig = MISSING
 
 
-DATASET_NAME = "argoverse-v1.1"
+@dataclass(frozen=True)
+class ArgoverseConfig(SensorDatasetConfig):
+    RING_CAMERA_HEIGHT: int = 1200
+    RING_CAMERA_WIDTH: int = 1920
+
+    STEREO_CAMERA_HEIGHT: int = 2056
+    STEREO_CAMERA_WIDTH: int = 2464
+
+    dataset_name: str = "argoverse"
```

Comment:
i don't think this is specific enough to be helpful. if it was "argoverse-tracking" or "argoverse-v1.1", perhaps.

Reply:
i vote we keep it as before
```
  dataset_name: argoverse-v1.1
```

Reply:
👍